nix/packages.nix

```diff
@@ -0,0 +1,44 @@
+{ mkYarnPackage, stdenv, lib, python, nodejs, cardano-explorer-app-src }:
+
+{
+  cardano-explorer-app = mkYarnPackage {
+    pname = "cardano-explorer-app";
```

Comment:
Ideally the name and version should be sourced from [package.json](https://github.com/input-output-hk/cardano-explorer-app/blob/develop/package.json)

Reply:
resolved with commit 09be18d